synapse/rest/client/v2_alpha/account.py

```diff
@@ -520,27 +682,31 @@ def on_POST(self, request):
                 400, "Missing param three_pid_creds", Codes.MISSING_PARAM
             )
 
+        # We ignore the bind parameter as this endpoint no longer is able to bind threepids
+        # to an identity server
+
         requester = yield self.auth.get_user_by_req(request)
         user_id = requester.user.to_string()
 
-        # Specify None as the identity server to retrieve it from the request body instead
-        threepid = yield self.identity_handler.threepid_from_creds(None, threepid_creds)
+        threepid_creds = body["three_pid_creds"]
+        assert_params_in_dict(threepid_creds, ["client_secret", "sid"])
 
-        if not threepid:
-            raise SynapseError(400, "Failed to auth 3pid", Codes.THREEPID_AUTH_FAILED)
-
-        for reqd in ["medium", "address", "validated_at"]:
-            if reqd not in threepid:
-                logger.warn("Couldn't add 3pid: invalid response from ID server")
-                raise SynapseError(500, "Invalid response from ID Server")
+        client_secret = body["client_secret"]
```

Comment:
isn't this and sid supposed to be in the threepid_creds, not the top of the body?